java-generator/core/src/test/resources/io/fabric8/java/generator/approvals/ApprovalTest.generate_withValidCrd_shouldGeneratePojos.testAkkaMicroservicesCrd.approved.txt

```diff
@@ -21,7 +21,7 @@ public class AkkaMicroserviceSpec implements io.fabric8.kubernetes.api.model.Kub
     @io.fabric8.generator.annotation.Pattern("^(off)$|^(102[4-9]|10[3-9][0-9]|1[1-9][0-9]{2}|[2-9][0-9]{3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$")
     @com.fasterxml.jackson.annotation.JsonPropertyDescription("Akka Management (Cluster Bootstrap) HTTP port in the range 1024 - 65535. Can be disabled with \"off\". This port is exposed as HTTP_MGMT_PORT environment variable if it is enabled.")
     @com.fasterxml.jackson.annotation.JsonSetter(nulls = com.fasterxml.jackson.annotation.Nulls.SKIP)
-    private String akkaManagementPort = io.fabric8.kubernetes.client.utils.Serialization.unmarshal("\"8558\"", String.class);
+    private String akkaManagementPort = "8558";
```

Comment:
This is better and absolutely fine for primitive values. But for the values object or array values that actually need to be parsed using `Serialization.unmarshal()`, why not store the parsed value in a static field so that we don't have to parse every time we create an instance?

Reply:
thanks for the suggestion @adriansuarez , it makes sense, I will integrate the necessary change next time I pick this up 👍

Reply:
done in the last iteration, let me know if the implementation aligns with what you had in mind.

Reply:
Perfect! Thanks for making this improvement.